lib/client.ts

```diff
@@ -11,6 +11,25 @@ const CircuitBreaker = require('circuit-breaker-js')
  * A request filter may introduce one or both functions in this interface. For more information
  * regarding request filters, refer to the readme of this project.
  */
+
+/**
+ * Defines options used by circuit-breaker-js
+ */
+export interface CircuitBreakerOptions {
+  windowDuration?: number
+  numBuckets?: number
+  timeoutDuration?: number
+  errorThreshold?: number
+  volumeThreshold?: number
+}
+
+/**
+ * This interface defines factory function for getting a circuit breaker
+ */
+export interface CircuitBreakerFactory {
+  (params: ServiceClientRequestOptions): any
```

Comment:
Shouldn't this return a circuit breaker?

Reply:
It should :) we're missing some types (I'd do this in another PR and mb event send types to DT...)